src/alchemlyb/workflows/abfe.py

```diff
@@ -0,0 +1,700 @@
+from os.path import join
+from glob import glob
+import pandas as pd
+import numpy as np
+import logging
+
+from ..parsing import gmx
+from ..preprocessing.subsampling import statistical_inefficiency
+from ..estimators import MBAR, BAR, TI
+from ..visualisation import (plot_mbar_overlap_matrix, plot_ti_dhdl,
+                             plot_dF_state, plot_convergence)
+from ..postprocessors.units import get_unit_converter
+from .. import concat
+
+
+class ABFE():
+    '''Alchemical Analysis style automatic workflow.
```

Comment:
The documentation does not make clear if this is a general workflow class or specific for one type of free energy calculations.

Reply:
Sorry. I use this workflow for my ABFE and RBFE calculations and it will work for these two types of calculations but I think the general workflow should be similar for other types of calculations?